ui/main/src/app/utils/TypeOfStateUtil.ts

```
@@ -0,0 +1,41 @@
+                if (allResponsesProvided && !hasAlarmSeverity) {
+                    return 'green';
+                }
+                if (allResponsesProvided && hasAlarmSeverity) {
+                    return 'red';
+                }
+                return 'darker-orange';
+            default:
+                return 'grey';
+        }
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)